integration-tests/README.md

```diff
@@ -1,3 +1,10 @@
 # integration-tests Project
 
-WIP
\ No newline at end of file
+Module contains integration tests verifying end to end functionality in real cluster. The tests are executed using command `mvn clean verify -Pintegration'.
+
+Prerequisites for running the tests:
+    - Cluster is up and running 
+    - All components are deployed using kustomize overlays 
+    - Configure minikube ip in 'application.properties' file for 'event-bridge.manager.url' and 'key-cloak.url' fields
```

Comment:
'event-bridge.manager.url' and 'key-cloak.url' should be defined as a Maven parameters. That way different values can be passed without changing code.

Reply:
Sure will move this configuration as Maven Parameters